assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/UserCountGraph.js

```diff
@@ -77,6 +78,9 @@ export default function UserCountGraph( { dimensionName, dimensionValue } ) {
 	const { startDate, endDate } = useSelect( ( select ) => select( CORE_USER ).getDateRangeDates( {
 		offsetDays: DATE_RANGE_OFFSET,
 	} ) );
+	const { getValue } = useSelect( ( select ) => select( CORE_FORMS ) );
+	const dimensionColor = getValue( FORM_ALL_TRAFFIC_WIDGET, 'dimensionColor' );
+	const graphLineColor = dimensionColor || '#1a73e8';
```

Comment:
Let's write it in a single line:

```suggestion
	const graphLineColor = useSelect( ( select ) => select( CORE_FORMS ).getValue( FORM_ALL_TRAFFIC_WIDGET, 'dimensionColor' ) || '#1a73e8' );
```